src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -15,7 +15,7 @@
     <!-- Disable native AOT on FreeBSD when cross building from Linux. -->
     <NativeAotSupported Condition="'$(TargetOS)' == 'freebsd' and '$(CrossBuild)' == 'true'">false</NativeAotSupported>
     <PublishAot Condition="'$(NativeAotSupported)' == 'true'">true</PublishAot>
-    <SysRoot Condition="'$(NativeAotSupported)' == 'true' and '$(CrossBuild)' == 'true' and '$(HostOS)' != 'windows'">$(ROOTFS_DIR)</SysRoot>
+    <SysRoot Condition="'$(NativeAotSupported)' == 'true' and '$(CrossBuild)' == 'true' and '$(HostOS)' != 'windows' and $(OutputRID.StartsWith('tizen')) != 'true'">$(ROOTFS_DIR)</SysRoot>
```

Comment:
The condition is becoming more complex and it is dupicated in 4 places. Should we move it to more central location to deduplicate it?

Reply:
I decided to remake the solution to use `NativeAotSupported` condition on the advice of @sbomer and not to complicate  the state of the `SysRoot` condition.